src/views/tickets/orders-rsvp.php

```diff
@@ -20,74 +20,90 @@
+			</p>
 		</div>
-		<p class="reserved-by">
-			<?php printf( esc_html__( 'Reserved by %s', 'event-tickets' ), esc_html( $first_attendee['purchaser_name'] ) ); ?>
-			<?php printf( esc_html__( ' on %s', 'event-tickets' ), date_i18n( 'F j, Y', strtotime( esc_attr( $first_attendee['purchase_time'] ) ) ) ); ?>
-		</p>
-	</div>
 		<ul class="tribe-rsvp-list">
-		<?php foreach ( $attendees as $i => $attendee ): ?>
-			<?php $key = $attendee['order_id']; ?>
-			<li class="tribe-item<?php echo $view->is_rsvp_restricted( $post_id, $attendee['product_id'] ) ? 'tribe-disabled' : ''; ?>" <?php echo $view->get_restriction_attr( $post_id, $attendee['product_id'] ); ?> id="attendee-<?php echo $attendee['order_id']; ?>">
-				<p class="list-attendee"><?php printf( esc_html__( 'Attendee %d', 'event-tickets' ), $i + 1 ); ?></p>
-				<div class="tribe-answer">
-					<!-- Wrapping <label> around both the text and the <select> will implicitly associate the text with the label. -->
-					<!-- See https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly -->
-					<label>
-						<?php esc_html_e( 'RSVP: ', 'event-tickets' ); ?>
-						<?php $view->render_rsvp_selector( "attendee[{$key}][order_status]", $attendee['order_status'], $post_id, $attendee['product_id'] ); ?>
-					</label>
-				</div>
-				<div class="attendee-meta-row">
-					<?php
-					$meta_fields = Tribe__Tickets_Plus__Main::instance()->meta()->get_meta_fields_by_ticket( $attendee['product_id'] );
-					$meta_data = get_post_meta( $attendee['attendee_id'], Tribe__Tickets_Plus__Meta::META_KEY, true );
-					?>
-					<?php
-					foreach ( $meta_fields as $field ) {
-						if ( 'checkbox' === $field->type && isset( $field->extra['options'] ) ) {
-							$values = array();
-							foreach ( $field->extra['options'] as $option ) {
-								$key = $field->slug . '_' . sanitize_title( $option );
+			<?php foreach ( $attendee_group as $i => $attendee ): ?>
+				<?php $key = $attendee['order_id']; ?>
+				<li class="tribe-item<?php echo $view->is_rsvp_restricted( $post_id, $attendee['product_id'] ) ? 'tribe-disabled' : ''; ?>" <?php echo $view->get_restriction_attr( $post_id, $attendee['product_id'] ); ?> id="attendee-<?php echo $attendee['order_id']; ?>">
+					<p class="list-attendee"><?php printf( esc_html__( 'Attendee %d', 'event-tickets' ), $i + 1 ); ?></p>
+					<div class="tribe-answer">
+						<!-- Wrapping <label> around both the text and the <select> will implicitly associate the text with the label. -->
+						<!-- See https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly -->
+						<label>
+							<?php esc_html_e( 'RSVP: ', 'event-tickets' ); ?>
+							<?php $view->render_rsvp_selector( "attendee[{$key}][order_status]", $attendee['order_status'], $post_id, $attendee['product_id'] ); ?>
+						</label>
+					</div>
+					<?php if ( class_exists( 'Tribe__Tickets_Plus__Main' ) ): ?>
```

Comment:
+1 for adding the class_exists() guard!

It does make me think that the code within the block shouldn't be there (I feel it would be preferable to keep ET+ code in ET+ and we could achieve the same thing with an action hook) but I also realize that the code inside the block pre-dates your changes.